liblxqt-config-cursor/translations/lxqt-config-cursor_hr.ts

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="hr_HR">
+<context>
+    <name>SelectWnd</name>
+    <message>
+        <location filename="../ui/selectwnd.ui" line="14"/>
+        <source>LXQt Mouse Theme Configuration</source>
+        <translation>Konfiguracija LXQt teme miša</translation>
+    </message>
+    <message>
+        <location filename="../ui/selectwnd.ui" line="20"/>
+        <source>Select the cursor theme you want to use (hover preview to test cursor). &lt;b&gt;LXQt session needs restart after this change&lt;/b&gt;:</source>
+        <translation>Odaberite temu pokazivača koju želite koristiti (pređite preko pregleda da biste testirali pokazivač). &lt;b&gt;LXQt sesiju morate ponovno pokrenuti nakon ove promjene&lt;/b&gt;:</translation>
+    </message>
+    <message>
+        <location filename="../ui/selectwnd.ui" line="81"/>
+        <source>&amp;Install New Theme...</source>
+        <translation>&amp;Instaliraj novu temu...</translation>
+    </message>
+    <message>
+        <location filename="../ui/selectwnd.ui" line="88"/>
+        <source>&amp;Remove Theme</source>
+        <translation>&amp;Ukloni-temu</translation>
+    </message>
+    <message>
+        <location filename="../selectwnd.cpp" line="208"/>
+        <source>XCurTheme error</source>
+        <translation>Greška XCurTheme </translation>
+    </message>
+    <message>
+        <location filename="../selectwnd.cpp" line="209"/>
+        <source>You can&apos;t remove active theme!</source>
+        <translation>Ne možete ukoniti aktivnu temu!</translation>
+    </message>
+</context>
+<context>
+    <name>WarningLabel</name>
+    <message>
+        <location filename="../ui/warninglabel.ui" line="14"/>
+        <source>Form</source>
+        <translation>Forma</translation>
+    </message>
+    <message>
+        <location filename="../ui/warninglabel.ui" line="44"/>
+        <source>LXQt could not find any cursor theme. The default X11 cursor theme will be used instead. LXQt looked in the following directories:</source>
+        <translation>LXQt ne može naći nijednu temu pokazivača. Biti će korištena zadana X11 tema pokazivača.LXQt je gledao u slijedećedirektorije:     </translation>
```

Comment:
A space is missing between second and third sentence and there are unnecessary trailing spaces.